wp-content/themes/reconasia/assets/_scss/pages/archive.scss

```diff
@@ -1,6 +1,96 @@
 @use '../abstracts' as *;
 @use '../components/entry-header';
 
+.archive {
+  &__featured {
+    position: relative;
+    margin: 0 calc(var(--container-padding) * -1);
+    padding: 0 var(--container-padding);
+    background: $color-bg-dark-200;
+    @include shadow('lg');
+
+    @include breakpoint('small') {
+      margin: 0 auto;
+      padding: 0;
+    }
+
+    @include breakpoint('medium') {
+      background: transparent;
+      box-shadow: none;
+    }
+
+    &::after {
+      content: '';
+      position: absolute;
+      top: -20px;
+      right: calc(var(--container-padding) * -1);
+      width: calc(100% - var(--container-padding));
+      height: 50px;
+      background: #fff;
+    }
+
+    &-label {
+      position: relative;
+      z-index: 1;
+      display: flex;
+      align-items: flex-end;
+      width: fit-content;
+      height: rem(30);
+      margin-left: 0;
+      padding: rem(12) rem(32) 0 0;
+      color: $color-white-190;
+      @extend %text-style-label-small;
+      text-transform: uppercase;
+      background-color: $color-bg-dark-200;
+      @include shadow('lg');
+      clip-path: polygon(0 0, 85% 0, 100% 101%, 0% 101%);
+
+      @include breakpoint('small') {
+        padding-left: rem(24);
+      }
+    }
+
+    .post-block {
+      margin-top: rem(0);
+      margin-bottom: rem(16);
+      padding-top: rem(16);
+      background: $color-bg-dark-200;
+      border-bottom: 0 !important;
+
+      &.has-post-thumbnail {
+        @include breakpoint('medium') {
+          grid-template-areas:
+            'img title'
+            'img excerpt'
+            'img authors'
+            'img date'
+            'img categories';
+          grid-template-columns: 1fr 2fr;
+          grid-template-rows: max-content max-content max-content max-content auto;
+          column-gap: rem(32);
+        }
+      }
+
+      @include breakpoint('small') {
+        // padding-top: rem(16);
+        padding-right: rem(24);
+        padding-bottom: rem(24);
+        padding-left: rem(24);
+      }
+
+      @include breakpoint('medium') {
+        @include shadow('lg');
+      }
+
+      @include breakpoint('large') {
+        margin-bottom: rem(8);
+        padding: rem(48);
+        @include shadow('lg');
```

Comment:
We don't need to include this again, since it's being set at the medium breakpoint.